userspace/libsinsp/sinsp.cpp

```diff
@@ -1964,6 +1942,8 @@ void sinsp::get_capture_stats(scap_stats* stats) const
 {
 	/* On purpose ignoring failures to not interrupt in case of stats retrieval failure. */
 	scap_get_stats(m_h, stats);
+	stats->n_suppressed = m_suppress.get_num_suppressed_events();
```

Comment:
I can't say i like it (the fact that `scap_stats` defines a structure where 2 fields are now filled/owned by itself) but i understand this is the only way to achieve this cleanup.
Let's say that, hoping that in the future libscap won't exist anymore, this is a good step in that direction :)